packages/mobile/src/screens/product-info/product-info.tsx

```diff
@@ -67,7 +67,11 @@ const ProductInfo: FC = () => {
   }, [id, user, dispatch]);
 
   if (!product) {
-    return <Spinner />;
+    return (
+      <View style={[globalStyles.flex1, globalStyles.justifyContentCenter]}>
+        <Spinner />
+      </View>
+    );
```

Comment:
You can just use - `return <Spinner isOverflow={true} />`

Reply:
fixed